src/st/sel/stream.c

```diff
@@ -1,37 +1,28 @@
 #include "sel.h"
 
-extern s32 g_StreamWidth;
-extern int g_StreamHeight;
-extern u32 g_StreamEndFrame;
-extern u32 g_StreamIsRGB24;
-extern s32 g_StreamRewindSwitch[];
-
-typedef struct {
-    u_long* vlcbuf[2];
-    int vlcid;
-    u_short* imgbuf[2];
-    int imgid;
-    RECT rect[2];
-    int rectid;
-    RECT slice;
-    int isdone;
-} DECENV;
-
-s32 func_801B9744(void) {
+// Check if CD data is ready as long as its not related to reading the position of the laser. The status is cached in
+// the `g_StreamDiskIsReady` global.
+//
+// returns:
+//     0 - disc isn't ready for a request
+//     1 - disc is ready
+s32 StreamDiskIsReady() {
```

Comment:
Very minor nitpick but you're using "Disk" in the function name, and "disc" in the comments with the return info. I believe "Disc" is preferred for optical media, but either way doesn't matter :)

Reply:
Looking through the PSY-Q docs (which are new to me), Sony used "disk" in their API names and documentation, though I completely agree it should be "disc". I'll update the comment for consistency.